### PR TITLE
Add Static Form with steps functionality

### PR DESCRIPTION
**Pull request by abedshamia**
https://user-images.githubusercontent.com/73759748/168493596-24bd036b-19d1-44a8-8a42-4589d9283083.mp4

### CODE REVIEW

File: client/src/components/Forms/Interview/Steps/StepFive.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <p>{title}</p>
+      </div>
+      <div className="interview-step-content">
+        <div className="interview-form__radio-group-fifth">
+          <div className="interview-form__radio-group-fifth-check">
+            <CheckCircleTwoTone
+              twoToneColor="#52c41a"
+              className="interview-form__radio-group-fifth-check-icon"
+            />
+            <Typography.Title>
```
Comment (Raghad-Mezied) [inline client/src/components/Forms/Interview/Steps/StepFive.jsx:28]: Destructure the title.

File: client/src/components/Forms/Interview/InterviewForm.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    setFormData({ ...formData, step: step - 1 });
+    setProgressPercent(25 * (step - 1));
+  };
+  const getAvailableTime = async () => {
+    if (!formData.specialization || !formData.language || !formData.questionCategory) {
+      message.error('Please select specialization, language and question category');
+      return;
+    }
+    try {
+      const { data: { data } } = await axios.post('/api/interview/available', {
```
Comment (Raghad-Mezied) [inline client/src/components/Forms/Interview/InterviewForm.js:46]: Why is this route POST, don't we want to get the available dates?

File: client/src/components/Forms/Interview/Hours/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/prop-types */
+// eslint-disable-next-line react/prop-types
```
Comment (Raghad-Mezied) [inline client/src/components/Forms/Interview/Hours/index.js:2]: Fix all `eslint` problems

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

File: client/src/components/Forms/Interview/InterviewForm.js [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+import {
+  StepOne, StepTwo, StepThree, StepFour,
+} from './Steps';
+import './InterviewForm.css';
+import StepFive from './Steps/StepFive';
+
+const { Step } = Steps;
+const InterviewForm = () => {
+  const [visible, setVisible] = React.useState(true);
+  const [formData, setFormData] = React.useState({
```
Comment (Raghad-Mezied) [inline client/src/components/Forms/Interview/InterviewForm.js:16]: You already destructured the useState, so just use it.

File: client/src/components/Forms/Interview/InterviewForm.js [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+  useEffect(() => {
+    if (loading) {
+      const doneBtn = document.getElementById('done');
+      if (doneBtn) doneBtn.disabled = true;
+    } else {
+      const doneBtn = document.getElementById('done') ? document.getElementById('done') : null;
+      if (doneBtn) doneBtn.disabled = false;
+    }
+  }, [loading]);
```
Comment (Raghad-Mezied) [inline client/src/components/Forms/Interview/InterviewForm.js:39]: Why you are manipulating dom here? you have a state and you can directly disable the btn according to your state.

File: client/src/components/Forms/Interview/InterviewForm.js [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+      nextStep();
+      const { data: { data } } = await axios.post('/api/interview/available', {
+        specialization: formData.specialization,
+        language: formData.language,
+      });
+      setAvailableTime(data);
+      setLoading(false);
+    } catch (err) {
+      message.error(err.response.data.message);
+    }
```
Comment (Raghad-Mezied) [inline client/src/components/Forms/Interview/InterviewForm.js:66]: Destructure the message

File: client/src/components/Forms/Interview/InterviewForm.js [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+    if (!formData.date || !formData.time || !formData.interviewerId) {
+      message.error('Please select time and date');
+      return;
+    }
+    try {
+      setLoading(true);
+      nextStep();
+      const { step: stepIgnored, ...rest } = formData;
+      const response = await axios.post('/api/interview', {
```
Comment (Raghad-Mezied) [inline client/src/components/Forms/Interview/InterviewForm.js:82]: Destructure the message

File: client/src/components/Forms/Interview/RadioInput/index.js [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+      name={name}
+      type="radio"
+      id={option}
+      value={option}
+      className="interview-form__radio-input"
+      onClick={handleChange}
+      checked={value === option}
+    />
+    <label htmlFor={option}>
+      {option === 'JS' ? 'JavaScript' : option}
```
Comment (Raghad-Mezied) [inline client/src/components/Forms/Interview/RadioInput/index.js:19]: Why this condition? if you want you can edit the database.

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

**Review by Mahmoud-Ahmad2 — APPROVED** [review]

**Review by Raghad-Mezied — APPROVED** [review]

**Review by manar-abed — APPROVED** [review]

**Pull request merged**